anitya/lib/backends/pypi.py

```diff
@@ -76,3 +78,30 @@ def get_versions(cls, project):
             raise AnityaPluginException('No JSON returned by %s' % url)
 
         return data['releases'].keys()
+
+    @classmethod
+    def check_feed(cls):
+        ''' Return a generator over the latest 40 uploads to PyPI
+
+        by querying an RSS feed.
+        '''
+
+        url = 'https://pypi.python.org/pypi?%3Aaction=rss'
+
+        try:
+            response = cls.call_url(url)
+        except Exception:  # pragma: no cover
+            raise AnityaPluginException('Could not contact %s' % url)
+
+        try:
+            parser = xml2dict.XML2Dict()
```

Comment:
I quite like this approach (rss to dict) and I wonder if it wouldn't make sense to port the SourceForge backend to use it